proposals/stf-general-investment.md

```diff
@@ -0,0 +1,263 @@
+    - [Warren](https://warren.io), a European self-service cloud computing platform
+    - [Google IDX](https://idx.dev), collaborative software development platform
+    - [Replit](https://replit.com)
+
+## Which target groups does your project address (who are its users?) and how would they benefit from activities proposed for funding (directly and indirectly)? 300 words
+
+The Nix ecosystem serves anyone who needs computers to do exactly as intended, repeatably, far into the future.
+Currently this manifests in three main target audiences:
+- Professional software developers and students of the field
+- Scientific researchers in industry and academia
+- Enterprises and organisations
+
+The proposed improvements for our contributor experience and quality of maintenance will benefit our users by:
+- Helping their projects to avoid bitrot, preserving the value of past investments
+- Making more dependencies readily available and keeping them up to date
+- Making for a safer, more robust software supply chain
+- Further ease self-hosting and customisation of software infrastructure
+- Leveraging open source, adopting improvements created by users in a sustainable way
+
+This will increase their productivity, and repeatability and portability of their work, and massively reduce the total cost of adoption and ownership of IT systems for everyone.
+
+## Please describe a specific scenario for the use of your technology and how this meets the needs of your target groups. 300 words
+
+In the course of their work, all of our target audiences may regularly need to:
+- Try out new technology by running programs without disrupting or subtly changing their working setup 
+
+  Nix, Nixpkgs, and NixOS support this workflow as a first class use case, and are further supported by an ecosystem of convenience tooling.
+
+- Develop software using pre-existing tools or libraries, possibly using multiple programming languages
+
+  Nixpkgs abstracts over the build process of numerous programming ecosystems, allowing to routinely and seamlessly integrate heterogenous environments.
+
+- Share complex setups with collaborators
+
+  The Nix ecosystem makes it trivial to create reproducible, portable development environments that work the same on each supported system.
+  With Nix, "works on my machine" is already "works everywhere, any time".
+
+- Test distributed systems
+
+  [NixOS virtual machine testing](https://nix.dev/tutorials/nixos/integration-testing-using-virtual-machines) is a highly convenient technique to ensure that networked applications work as intended, ubiquitous in the Nix ecosystem.
+
+- Create executables for multiple platforms
+
+  By transparently supporting cross-compilation, Nixpkgs eliminates almost all of the effort for targeting the most common system types.
+
+- Deploy their products or services with confidence
+
+  The Nix ecosystem promotes [the purely functional software deployment model](https://edolstra.github.io/pubs/phd-thesis.pdf).
```

Comment:
@edolstra It would be super nice to have a link to this document using a DOI. (In reference to the chat we had on PM on Matrix)